backend/config.js

```diff
@@ -0,0 +1,11 @@
+const config = {
+  environment: process.env.NODE_ENV || 'dev',
+  server: {
+    port: process.env.PORT || 8080
+  },
+  mongo: {
+    url: process.env.MONGO_DB_URI || 'mongodb://localhost/people'
```

Comment:
What sets `process.env.MONGO_DB_URI`? It prints out `undefined` when I log it. The only thing I could find about it when searching was [this heroku article](https://devcenter.heroku.com/articles/mongolab#getting-your-connection-uri). If we are using herko shouldn't this be `process.env.MONGODB_URI`? Unless I'm missing something here.

Reply:
nothing currently, i just scaffolded this part of the project and hadn't done anything with it. you're not crazy i'm just lazy